cypress/fixtures/plugins/security-dashboards-plugin/indexpatterns/indexPatternTenantHeader.json

```diff
@@ -0,0 +1,3 @@
+{
+  "securitytenant": ["test"]
+}
```

Comment:
Please add an empty line at the end of the file for all the new files.

Reply:
Just my 2 cents, we should enforce this using eslint instead of relying on PR's to catch it.

Reply:
Fixed, I also fixed the other files. Ya, I agree with Ashwin that we should apply eslint on this repo.